docker-compose.dev.yml

```diff
@@ -21,6 +23,8 @@ services:
     volumes:
     - ./samples:/mnt/samples
     - ./src:/usr/src/app/src
+    depends_on:
+      - "redis"
```

Comment:
Doesn't dev-web also depend on ursadb?

Reply:
Right, for some reason I've assumed that it passes everything to daemon.